mydoc/mydoc_cmip6_user_input.md

```diff
@@ -86,17 +86,28 @@ permalink: /mydoc_cmip6_user_input/
+* *output_path_template*:    "\<mip_era\>\<activity_id\>\<institution_id\>\<source_id\>\<experiment_id\>\<member_id\>\<table\>\<variable_id\>\<grid_label\>\<version\>",
 
-* *output_file_template*:    "<variable_id><table><source_id><experiment_id><member_id><grid_label>[<time_range>]",
+* *version*: Set this to name the \<version\> part of the output path template.  If not set, then CMOR will use the current date as the version in the format of "vYYYYMMDD" (ex. "v20220718"),
 
-* *license*:                "One of 2 licenses: ----- CMIP6 model data produced by <Your CentreName> is licensed under a Creative Commons Attribution 'NonCommercial Share Alike' 4.0 International License (http://creativecommons.org/licenses/by/4.0/). Use of the data should be acknowledged following guidelines found at <what URL???> The data is hosted via the Earth System Grid Federation. Permissions beyond the scope of this license may be available at http://pcmdi.org/cmip5/terms-of-use.   Individuals using this data should register at ??? to receive notice of selected categories of errata and updates. Further information about this data, including some limitations, can be found at ???. The data producers and data providers make no warranty, either express or implied, including but not limited to, warranties of merchantability and fitness for a particular purpose. All liabilities arising from the supply of the information (including any liability arising in negligence) are excluded to the fullest extent permitted by law. "
+* *output_file_template*:    "\<variable_id\>\<table\>\<source_id\>\<experiment_id\>\<member_id\>\<grid_label\>[\<time_range\>]",
+
+* *license*: Use the following template
+    * "CMIP6 model data produced by **\<institution's name\>** is licensed under a **\<license\>**. Consult https://pcmdi.llnl.gov/CMIP6/TermsOfUse for terms of use governing CMIP6 output, including citation requirements and proper acknowledgment. Further information about this data, including some limitations, can be found via the further_info_url (recorded as a global attribute in this file)**[ and at \<URL\>]**. The data producers and data providers make no warranty, either express or implied, including, but not limited to, warranties of merchantability and fitness for a particular purpose. All liabilities arising from the supply of the information (including any liability arising in negligence) are excluded to the fullest extent permitted by law."
+
+    * **[ and at \<URL\>]** is a placeholder for an optional URL for information on the data.
+
+    * Choose one of the following for **\<license\>**
+        * Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/)
+        * Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License (https://creativecommons.org/licenses/by-nc-sa/4.0/)
```

Comment:
We want to limit the use of the more restrictive licenses, so I would remove the CC BY-NA-SA 4.0 and CC BY-SA 4.0 variants from this doc to limit their usage propagation. We could always point to the https://github.com/WCRP-CMIP/CMIP6_CVs/blob/master/CMIP6_license.json for a CMIP6 demo

Reply:
The licenses have been removed.